delivery/meteorController.js

```diff
@@ -15,9 +15,12 @@ meteorRouter.get('/meteors', async (req, res) => {
     ({ startDate, endDate } = setDefaultDates(startDate, endDate));
 
     const meteorFileteredData = await getMeteorFilteredData(startDate, endDate, Boolean(count), Boolean(wereDangerousMeteors));
-    res.json({
+
+    res.render('../views/meteors.njk', {
       message: `Meteors observed from ${startDate} to ${endDate}`,
-      data: meteorFileteredData
+      meteorData: meteorFileteredData.meteorData,
+      wereDangerousMeteors: meteorFileteredData.wereDangerousMeteors,
+      count: meteorFileteredData.count
     });
   } catch (error) {
     res.status(500).json({ error: error.message });
```

Comment:
you can find some advice related to error handler here https://teams.microsoft.com/l/message/19:6a563893960e4d50a159c3859a776a7d@thread.v2/1728559851120?context=%7B%22contextType%22%3A%22chat%22%7D